Basic/APIDiorama/Assets/Editor/Tutorials/TableOfContent.asset

```diff
@@ -0,0 +1,73 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 89305aa391d1c5141bbe1628d930a2c5, type: 3}
+  m_Name: TableOfContent
+  m_EditorClassIdentifier: 
+  Modified:
+    m_PersistentCalls:
+      m_Calls: []
+  ParentContainer: {fileID: 0}
+  OrderInView: 0
+  BackgroundImage: {fileID: 0}
+  Title:
+    m_Untranslated: Table of Content
```

Comment:
```suggestion
    m_Untranslated: Table of Contents
```

Reply:
I'm not sure if "Table of Content" is specific to IETs, but it seems like a typo 😅

Reply:
After misspelling it for 2 years, today I learned that it's table of contentS, thanks Jil <3